ccd-definition/CaseEventToComplexTypes/draft-CMO/allParties.json

```diff
@@ -0,0 +1,30 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "AllParties",
+    "CaseEventID": "draftCMO",
+    "CaseFieldID": "allParties",
+    "ListElementCode": "directionType",
+    "EventElementLabel": " ",
+    "FieldDisplayOrder": 1,
+    "DisplayContext": "OPTIONAL"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "AllParties",
+    "CaseEventID": "draftCMO",
+    "CaseFieldID": "allParties",
+    "ListElementCode": "directionText",
+    "FieldDisplayOrder": 2,
+    "DisplayContext": "OPTIONAL"
```

Comment:
Isn't this mandatory?

Reply:
Good question, I assumed from the AC it was all optional. Checking with BA.

Reply:
Description is mandatory, this will affect the above test also.